### PR TITLE
GHC-81995: add error message and example using `sequence`

### PR DESCRIPTION
**Pull request by dschrempf**

### CODE REVIEW

File: message-index/messages/GHC-81995/sequence/before/GetLine.hs [hunk 1]
```diff
@@ -0,0 +1,6 @@
+module GetLine where
+
+main :: IO ()
+main = do
+  sequence [putStrLn "First line", putStrLn "Second line"]
```
Comment (jllang) [inline message-index/messages/GHC-81995/sequence/before/GetLine.hs:5]: I think we could also have a pure example, e.g., if we define
```
list :: [Int]
list = do
  return 1
  return 2
```
we get `[2]` as the resulting value for `list`.

Comment (dschrempf) [reply message-index/messages/GHC-81995/sequence/before/GetLine.hs:5]: That's right. I can add another example. I think a classic example using IO is good to have.

Comment (dschrempf) [reply message-index/messages/GHC-81995/sequence/before/GetLine.hs:5]: @jilang I have added your nice example.

**Review by BinderDavid — APPROVED** [review]
Thanks for the contribution! And sorry for the delayed review.

**Comment by dschrempf** [comment]
No problem, and thanks for merging!

**Pull request merged**